mapdata/WarcraftLegacies/war3mapSkin.txt

```diff
@@ -12,37 +9,9 @@ InfoPanelIconArmorNormal=UI\Widgets\Console\Human\UPGGate.blp
 InfoPanelIconArmorNormalNeutral=UI\Widgets\Console\Human\BTNGate.blp
 
 [FrameDef]
-ARMORTIP_DIVINE="Pierce: |cff00ff0050%|r|NSiege: |cff00ff0050%|r|NMagic: |cff00ff0050%|r|nSpell: |cff00ff000%|r|NNormal: |cffffff00100%|r|NHero: |cffffff00100%|r|NChaos: |cffff0000110%|r"
```

Comment:
It looks like your export to MapData deleted all of our custom Game Interface stuff. I don't know why this happens unfortunately, it happens to me too sometimes. Need to just revert all changes to this file.